spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/config/GcpConfigBootstrapConfiguration.java

```diff
@@ -31,6 +31,7 @@
  * @author Jisha Abubaker
  * @author João André Martins
  * @author Mike Eltsufin
+ * @since 1.1
```

Comment:
Everything is good, but as we discussed before `@since` on class level should be separated with the blank line, but not stick to the `@author` list.
WDYT?

Reply:
I'll make the change ... no worries.

It seems this isn't consistent across Spring Java code, which makes it confusing.

Reply:
Yeah, agreed.

I guess eventually we all come up with this foundation: https://github.com/spring-io/spring-javaformat/